src/Pages/account/Account.js

```diff
@@ -2,22 +2,48 @@ import React, { Component } from 'react';
 import Input from '../../Components/input/Input';
 import Button from '../../Components/button/Button';
 import './Account.scss';
+import { Link } from 'react-router-dom';
+import Nav from '../../Components/nav/Nav';
+import Footer from '../../Components/footer/Footer';
 
 class Account extends Component {
-    render() {
+    state = {
+        email: '',
+        password: ''
+    };
+
+    handleOnclick = (e) => {
+        e.preventDefault();
+        fetch('http://10.58.5.213:8000/sign/in', {
```

Comment:
```suggestion
        fetch(`{API}/sign/in`, {
```
이런 주소는 루트 폴더에 config.js 만드시고 모아서 관리해주세요. 주소들이 여기저기 흩어져 있는 것보다 하나의 변수로 관리해야 나중에 ip 주소 변경시 대응하기 편합니다.

Reply:
넵 수정하겠습니다